reference/service/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Pulumi Cloud Console
+---
+
+[The Pulumi Cloud Console](https://app.pulumi.com) allows you to manage your stacks online. It enables
+you to to collaborate with multiple developers, protect against concurrent updates, store resource
+history, integrate with your CI/CD system, and more!
+
+## Getting Started
+
+To get started using the Pulumi Cloud Console, go to [app.pulumi.com](https://app.pulumi.com) and
+sign in with your GitHub account.
+
+> **NOTE**
+>
+> Support for more kinds of identity providers is coming soon. If you require something
+> other than GitHub, please [let us know](https://www.pulumi.com/about/#contact-us).
+
+## Organizations
+
+The Pulumi Cloud Console manages stacks by placing them within _Organizations_.
+
+When you first log in, a Pulum organization is created with the same name as your GitHub account.
+That organization is implied whenever you run commands like `pulumi stack init` or
+`pulumi stack select`. For example, `pulumi stack init my-devstack` will create a new stack in
+the organization associated with your GitHub account. e.g. `octocruise/my-devstack`.
+
+To create or select a stack in a different organization, prefix it with the organization's name
```

Comment:
Should we put this in the FAQ also? I.e.,

## How do I create a stack inside an Organization instead of my User account?

Reply:
Done.